Cargo.toml

```diff
@@ -22,20 +22,24 @@ rust-version = "1.63"
 
 [features]
 eh0 = ["dep:eh0", "dep:nb"]
-eh1 = ["dep:eh1", "dep:embedded-hal-nb"]
+eh1 = ["dep:eh1", "dep:embedded-hal-nb", "dep:embedded-io"]
 
 embedded-time = ["dep:embedded-time", "dep:void"]
-embedded-hal-async = ["dep:embedded-hal-async"]
+embedded-hal-async = ["dep:embedded-hal-async", "dep:embedded-io-async"]
 
 default = ["eh0", "embedded-time"]
 
 [dependencies]
-eh0 = { package = "embedded-hal", version = "0.2.7", features = ["unproven"], optional = true }
+eh0 = { package = "embedded-hal", version = "0.2.7", features = [
+    "unproven",
+], optional = true }
```

Comment:
This is a result of automatic formating on file save. If you could tell me if this file gets formated the same way for you I would be thankful. If it happens only on my machine I will revert this change.

Reply:
What editor and formatting plugin do you use? rustfmt does not reformat TOML files by default.

I would prefer one-line-per-dependency (I find it easier to read), so it would be nice if you could revert the change 🙂

Reply:
Yup, this is a result of formatting from "Even Better TOML" extension in VS Code. Reverting this change.